src/features/game/expansion/lib/boosts.ts

```diff
@@ -108,3 +108,24 @@ export const hasBoost = ({ item, collectibles }: HasBoostArgs) => {
 
   return false;
 };
+
+/**
+ * Get boosted exp from Bumpkin skills.
+ * Decimal mul for precision.
+ * @todo add "Curer" skill once "Fermenter Goods" are finalized
+ * @param foodExp value to be increased
+ * @param bumpkin to check for skills
+ * @returns boosted food exp
+ */
+export const getFoodExpBoost = (
+  foodExp: number,
+  bumpkin: Bumpkin | undefined
+): number => {
+  let boostedExp = new Decimal(foodExp);
+
+  if (bumpkin?.skills["Kitchen Hand"]) {
+    boostedExp = boostedExp.mul(1.1);
+  }
+
+  return boostedExp.toNumber();
```

Comment:
i think its better to return a decimal and work with it later.

Reply:
I tried to avoid changing the current `number` type declaration for bumpkin experience as it may affect other components i havent checked as well as backend stuff.

or is your suggestion to return Decimal and just call `toNumber()` after function call on each component that uses it?

Reply:
> I tried to avoid changing the current `number` type declaration for bumpkin experience as it may affect other components i havent checked as well as backend stuff.
> 
> or is your suggestion to return Decimal and just call `toNumber()` after function call on each component that uses it?

@j923sneaks if all bumpkin exps is a Number then just let code as it is. 

I think everything should be a Decimal, so we can avoid future errors, idk why it its a number, maybe a dev can explain to us.